GiEnJul/ClientApp/src/components/admin/suggestedConnections/ConnectionSuggesterMacro.tsx

```diff
@@ -130,13 +125,13 @@ const ConnectionSuggesterMacro: React.FC<ConnectionSuggesterMacro> = ({
           >
             Koble sammen
           </Button>
-          <p></p>
           <Button
             onClick={getSuggestedRecipients}
             fullWidth
             variant="contained"
             color="secondary"
             disabled={state.refreshing}
+            style={{ marginTop: "15px" }}
```

Comment:
Nitpick: Changing this into a relative length would make the page better for different screen sizes. https://www.w3schools.com/cssref/css_units.asp. 

I'm a bit unsure if inline css is best practice, as I think this can make the code a bit unreadable. Would it probably be better to use styling classes? If so, this could be applied elsewhere as well.